_posts/2022-07-04-Moderator-Applications.md

```diff
@@ -2,6 +2,7 @@
 title: Moderator Applications
 author: Brottweiler
 categories: announcement
+excerpt: Finally, a way for You to apply!
```

Comment:
```suggestion
excerpt: Finally, a way for you to apply!
```

Believe that'd be the proper grammar. Fix the issue in the post as well too, thanks!